org/pr/android-diff-size.ts

```diff
@@ -6,6 +6,8 @@ export default async () => {
     const modifiedFiles = danger.git.modified_files;
     const createdFiles = danger.git.created_files;
     const deletedFiles = danger.git.deleted_files;
+    const additions = danger.github.pr.additions;
+    const deletions = danger.github.pr.deletions;
```

Comment:
I think this could be worth a comment on line 5 before all those `const` to explain why this is important (i.e. workaround a glitch in Peril) and not just for style. And also to let others know that this Peril issue exist and that this is the workaround to solve it, for future us who were not aware and might need to apply it for other properties if that happens again in the future.

Reply:
Yup! Done in `c5e3b95`

Reply:
Perfect, thanks 👍